src/plugins/data/common/search/search_source/search_source_service.ts

```diff
@@ -61,6 +61,10 @@ export class SearchSourceService {
        * creates searchsource based on serialized search source fields
        */
       create: createSearchSource(indexPatterns, dependencies),
+      createLazy: (searchSourceFields: SerializedSearchSourceFields = {}) => {
+        const fn = createSearchSource(indexPatterns, dependencies);
+        return fn(searchSourceFields, true);
+      },
```

Comment:
if tried to add just a configuration to the create function which uses a DataViewLazy then, for whatever reason it didn't work. with a dedicated function it did. it's a POC, it doesn't need to be polished